samples/exitPool/exitPoolSample.py

```diff
@@ -34,17 +33,16 @@ def main():
         quit()
 
     print()
-    print("===============================================================")
-    print("================= Step 2: Send Bpt to Pool =================")
-    print("===============================================================")
+    print("==============================================================")
+    print("================== Step 2: Send Bpt to Pool ==================")
```

Comment:
Didn't catch this on the last review. I get what you mean conceptually, but it's more accurate to say "burn the BPT" though I get that may be confusing for the user.

What do you think about just `Step 2: Exit Pool`?